### DISCUSSION TITLE
Building a GUI over Dep-Scan CLI

### DISCUSSION THREAD

**Discussion by saketjajoo** [owasp-dep-scan/dep-scan#181] [Ideas]
The suggestion here is to build a GUI-like application over the CLI app (or convert the CLI into a GUI via a command line flag). More details are in the [GUI_README.md doc](https://github.com/owasp-dep-scan/dep-scan/blob/master/contrib/GUI_README.md).

Open to any comments/concerns/critiques/etc.

**Comment by prabhu** [comment]
How about an electron app that can render VDR/VEX and Reachable flows inline with working hyperlinks to the codebase for easy navigation?
